py/__init__.py

```diff
@@ -925,6 +925,10 @@ def line(self, Y, X=None, win=None, env=None, opts=None, update=None,
             else:
                 assert X is not None, 'must specify x-values for line update'
         assert Y.ndim == 1 or Y.ndim == 2, 'Y should have 1 or 2 dim'
+        assert Y.shape[-1] > 0, 'must plot one line at least'
+        if Y.ndim == 2 and Y.shape[1] == 1:
+                Y = Y.reshape(Y.shape[0])
+                X = X.reshape(X.shape[0])
```

Comment:
Ack! `X` could be `None` here. You should reshape `X` _after_ line 936. Perhaps move this if `Y` block to after there?

Reply:
Try
```
 win = viz.line(
        Y=np.column_stack((np.arange(0, 10), np.arange(0, 10))),
    )
```
and 
```
 win = viz.line(
        Y=np.arange(1, 10)
    )
```